keps/prod-readiness/sig-auth/2799.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2799
+alpha:
+  approver: "@deads2k"
```

Comment:
also beta since some one of the gates is proposing to start at beta?

Reply:
done